src/store/modules/user/UserState.ts

```diff
@@ -2,5 +2,7 @@ export default interface UserState {
     token: string;
     current: object | null;
     currentFacility: object | null;
+    currentFacilityLocation: object | null;
```

Comment:
We may be having already set facility location for shipments and purchase order. We need to rethink if this is to be persisted in state or we can just have the existing or first value from the list for empty case